[ObsUX] Fix bottom gap while loading data

c-georgakopoulos opened the pull request:
## Summary

Fixes #229666 

When user zooms-out below the 100% value, a gap appeared in the loading screen.

**BEFORE**:
<img width="2560" height="1315" alt="my-first-ticket-BEFORE" src="https://github.com/user-attachments/assets/f6777672-873f-4f2c-9292-c055e717e98b" />

<br>
<br>
 
**AFTER**: 
<img width="1297" height="1090" alt="Screenshot from 2025-10-16 18-34-57" src="https://github.com/user-attachments/assets/27e75c36-a568-4e71-9041-b0c669c00c52" />

<img width="1297" height="1090" alt="Screenshot from 2025-10-16 18-34-49" src="https://github.com/user-attachments/assets/cc50543f-53ba-4146-a3bc-45a2248bda0c" />


<!--ONMERGE {"backportTargets":["9.2"]} ONMERGE-->

c-georgakopoulos committed:
modified parent container height

c-georgakopoulos committed:
fixed problem for zoom-in and zoom-out situations. The container is presented correctly regardless of zoom value

c-georgakopoulos committed:
removed import that was imported by accident

c-georgakopoulos committed:
Merge branch 'main' into 229666-obsoverview-fix-bottom-gap-while-loading-data

kibanamachine committed:
[CI] Auto-commit changed files from 'node scripts/eslint_all_files --no-cache --fix'

kpatticha commented:
@c-georgakopoulos congrats on your first PR 🥳 

It would be helpful to update your description to include at least the following.
- Link the GitHub issue addressed by this PR
- Add screenshots with before and after state

miloszmarcinkowski commented on x-pack/solutions/observability/plugins/observability/public/components/loading_observability.tsx:
```diff
@@ -14,7 +14,11 @@ export function LoadingObservability() {
+    <ObservabilityPageTemplate
+      pageSectionProps={{ alignment: 'center' }}
+      showSolutionNav={false}
+      style={{ height: '100vh', position: 'fixed', inset: 0 }}
```
Congratulations on your first PR 🎉  

If you check `check_types.sh` job in the CI you will see that it failed due to an invalid `style` property.

In that case, I would suggest using `css` prop as advised [here](https://github.com/elastic/eui/discussions/6828#discussioncomment-7038434). Other than that, the solution looks good to me!


```suggestion
      css={css`
        height: 100vh;
        position: fixed;
        inset: 0;
      `}
```

c-georgakopoulos committed:
Merge branch 'main' into 229666-obsoverview-fix-bottom-gap-while-loading-data

c-georgakopoulos committed:
Update x-pack/solutions/observability/plugins/observability/public/components/loading_observability.tsx

Update prop: style -> css . This fixes the failed ci (failed due to improper typing)

Co-authored-by: Miłosz Marcinkowski <38698566+miloszmarcinkowski@users.noreply.github.com>

c-georgakopoulos committed:
Merge branch 'main' into 229666-obsoverview-fix-bottom-gap-while-loading-data

c-georgakopoulos committed:
import css

c-georgakopoulos committed:
Merge branch 'main' into 229666-obsoverview-fix-bottom-gap-while-loading-data

elasticmachine commented:
## :yellow_heart: Build succeeded, but was flaky
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/353001)
* Commit: 0c84aa4ef519fc1f256ec12b5743c8087d57c186

### Failed CI Steps
* [Scout: [ platform / discover_enhanced ] plugin](https://buildkite.com/elastic/kibana-pull-request/builds/353001#019a062a-3472-4fe3-8d86-e9b5370d8de5)

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/1bfd3c1046ae74295d77a33d68a0aa91071ce5e5) | [after](https://github.com/elastic/kibana/commit/0c84aa4ef519fc1f256ec12b5743c8087d57c186) | diff |
| --- | --- | --- | --- |
| `observability` | 1.7MB | 1.7MB | +76.0B |



### History
* :broken_heart: [Build #352482](https://buildkite.com/elastic/kibana-pull-request/builds/352482) failed 619bb810981f958f9a585545b2a73f526b24efb6
* :broken_heart: [Build #352459](https://buildkite.com/elastic/kibana-pull-request/builds/352459) failed 475fbfa872245073b2699770295efefdbe5054a6
* :broken_heart: [Build #351850](https://buildkite.com/elastic/kibana-pull-request/builds/351850) failed 2667ded1bd0903187f30c095206cbc597257e4af
* :broken_heart: [Build #350799](https://buildkite.com/elastic/kibana-pull-request/builds/350799) failed 855f0d7a1579ea473b74d482069acd0bd9099a2a

cc @c-georgakopoulos

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/353001","number":353001,"commit":"0c84aa4ef519fc1f256ec12b5743c8087d57c186"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/352482","number":352482,"commit":"619bb810981f958f9a585545b2a73f526b24efb6"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/352459","number":352459,"commit":"475fbfa872245073b2699770295efefdbe5054a6"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/351850","number":351850,"commit":"2667ded1bd0903187f30c095206cbc597257e4af"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/350799","number":350799,"commit":"855f0d7a1579ea473b74d482069acd0bd9099a2a"}],"number":353001}
buildkite-pr-comment-->

miloszmarcinkowski reviewed (approved):
Tested locally, LGTM!

iblancof reviewed (approved):
Tested in local and code changes LGTM!

c-georgakopoulos committed:
Merge branch 'main' into 229666-obsoverview-fix-bottom-gap-while-loading-data

c-georgakopoulos committed:
Merge branch 'main' into 229666-obsoverview-fix-bottom-gap-while-loading-data

c-georgakopoulos committed:
Merge branch 'main' into 229666-obsoverview-fix-bottom-gap-while-loading-data

c-georgakopoulos committed:
Merge branch 'main' into 229666-obsoverview-fix-bottom-gap-while-loading-data
